src/DsTrinityDataBundle/Service/Builder/ObjectListBuilder.php

```diff
@@ -33,11 +33,25 @@ public function __construct(Connection $db, EventDispatcherInterface $eventDispa
     /**
      * {@inheritdoc}
      */
-    public function buildByList(array $options): array
+    public function buildByList(array $options): \Generator
     {
-        $list = $this->getList($options);
-
-        return $list->getObjects();
+        if ($options['object_limit_per_class']) {
```

Comment:
`object_limit_per_class` is `0` by default, so no objects will be fetched by default.
Could you please change the statement so that the default behavior still works (no object limit per class)

Reply:
@solverat I will not have time in the coming weeks to work on this. You can either wait another couple of weeks, or we close it. It is open for almost 4 years now, so what are a couple more weeks right? :D